src/ProjectService.Data/ProjectRepository.cs

```diff
@@ -43,12 +43,10 @@ public IEnumerable<DbProjectUser> GetProjectUsers(Guid projectId, bool showNotAc
             return provider.ProjectsUsers.Include(u => u.Role).Where(predicate).ToList();
         }
 
-        public Guid CreateNewProject(DbProject newProject)
+        public void CreateNewProject(DbProject newProject)
```

Comment:
Why?

Reply:
What?

Reply:
Response: base project info

Reply:
Why? I do it in command

Reply:
Because it is written in the technical description